molgenis/molgenis-emx2
# chore(directory): Add hyperlink field to store DUC profiles

how to test:
- goto https://preview-emx2-pr-3474.dev.molgenis.org/directory-demo/tables/#/Collections, login and see that the DUC field is added.

todo:
- [na] updated docs in case of new feature
- [na] added tests
